### PR TITLE
(SIMP-2673) Tag the 6.0.0-Beta Release

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-2673 #close
SIMP-2672 #close

### CODE REVIEW

File: Puppetfile.stable [hunk 1]
```diff
@@ -2,370 +2,434 @@ moduledir 'src'
...
 mod 'simp-oddjob',
   :git => 'https://github.com/simp/pupmod-simp-oddjob',
-  :ref => '8c7cc73766936278e3f97ecb90525ce0307e20ee'
+  :ref => '0c79577630e86b6e2859738a04d65e4489ed29cd'
 
-mod 'simp-openldap',
-  :git => 'https://github.com/simp/pupmod-simp-openldap',
-  :ref => '9847a8f30a6bffef6db2afa92dc71bf6d1abd409'
+mod 'simp-simp_openldap',
+  :git => 'git@github.com:trevor-vaughan/pupmod-simp-simp_openldap',
```
Comment (jeefberkey) [inline Puppetfile.stable]: Testing mode

File: Puppetfile.stable [hunk 2]
```diff
@@ -2,370 +2,434 @@ moduledir 'src'
...
   :git => 'https://github.com/simp/pupmod-simp-simp',
-  :ref => '4b9e53ae884a39296b21482ccf3db066f8df6278'
+  :ref => 'e5cb0fae497e4fb84528796a7031f84afd59d85a'
+
+mod 'simp-simpcat',
+  :git => 'https://github.com/simp/pupmod-simp-simpcat',
+  :ref => '50deb1cecc783b28aed39e348da45cd212c37706'
+
+mod 'simp-simp_apache',
+  :git => 'git@github.com:trevor-vaughan/pupmod-simp-apache',
```
Comment (jeefberkey) [inline Puppetfile.stable]: Testing mode

File: Puppetfile.stable [hunk 3]
```diff
@@ -2,370 +2,434 @@ moduledir 'src'
...
+  :ref => '5ed90c390cdf72ccc10cbe1f79dcb08a17aed4cb'
+
+mod 'simp-simp_rsyslog',
+  :git => 'https://github.com/simp/pupmod-simp-simp_rsyslog',
+  :ref => 'bac200cbef242f08dfc3a570680667d2812e65c0'
 
 mod 'simp-simplib',
-  :git => 'https://github.com/simp/pupmod-simp-simplib',
-  :ref => 'aba8d486d8fa23a6903d6c327e55f6b48b68cb12'
+  :git => 'git@github.com:trevor-vaughan/pupmod-simp-simplib',
```
Comment (jeefberkey) [inline Puppetfile.stable]: Testing mode

File: Puppetfile.stable [hunk 4]
```diff
@@ -2,370 +2,434 @@ moduledir 'src'
...
+
+mod 'elastic-elasticsearch',
   :git => 'https://github.com/simp/puppet-elasticsearch',
   :ref => 'dc6d3fab6ace62701aa3de6e4bb868e24fef0553'
 
-mod 'puppet-logstash',
-  :git => 'https://github.com/simp/puppet-logstash',
-  :ref => 'd02c78b014be9310eabbbb7d8267475efcf70e07'
+mod 'elastic-logstash',
+  :git => 'git@github.com:trevor-vaughan/puppet-logstash',
```
Comment (kendall-moore) [inline Puppetfile.stable]: I believe this should be the public https URL.

**Pull request merged**